pyproject.toml

```diff
@@ -73,7 +73,8 @@ dependencies = [
   "pypatchmatch",
   "python-multipart",
   "requests",
-  "semver~=3.0.1"
+  "semver~=3.0.1",
+  "scikit-image"
```

Comment:
Reminder to remove `scikit-image` and run `uv lock` after removing the non-commercial OpenPose code